operation/fhir-operation-reindex/src/main/java/com/ibm/fhir/operation/reindex/ReindexOperation.java

```diff
@@ -92,12 +94,14 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
                             // assume full ISO format
                             tstamp = Instant.parse(val);
                         }
-
                     } else if (PARAM_RESOURCE_COUNT.equals(parameter.getName().getValue())) {
```

Comment:
We should probably have a defense against say 10000000 resources get reindexed. What's a reasonable defense/upper limit?

Reply:
done